README.md

```diff
@@ -33,67 +33,67 @@ Now let's take a look at these components individually.
 ### Nexus KnowledgeGraph
 This service is the heart of the Blue Brain Nexus platform. It allows users to define their domain, populate the knowledge graph with data, attach data files to entities. It also provides semantic search facilities to discover similar and relevant data in the platform.
 
-**Status**: active development
-**Source code**: https://github.com/BlueBrain/nexus-kg
+**Status**: active development  
+**Source code**: https://github.com/BlueBrain/nexus-kg  
```

Comment:
Is there a reason this whitespace was added here?

Reply:
It was intended, to get a line break.